blocks/card/card.css

```diff
@@ -0,0 +1,145 @@
+.card{
+    padding:20px 10px 60px;
+}
+.card__items{
+    list-style: none;
+    padding: 0;
+}
+.card__card{
+    padding: 20px 0;
+}
+.card__image{
+    position: relative;
+    background-repeat: no-repeat;
+    background-size: cover;
+    min-height:460px ;
+    border-radius: 20px;
+}
+.card__image_type_customers{
+    background-image: url(../../images/forCustomers_mobile.png);
+}
+.card__image_type_wedding{
+    background-image: url(../../images/weddings_mobile.png);
+}
+.card__image_type_group{
+    background-image: url(../../images/groupBook_mobile.png);
+}
+.card__image_type_corporate{
+    background-image: url(../../images/corporate_mobile.png);
+}
+.card__content{
+    position: absolute;
+    inset: 0;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    background-color: rgba(0, 0, 0, .5);
+    height: 100%;
+    border-radius: 20px;
+}
+.card__heading{
+    color: #fff;
+    font-family: 'Roboto Condensed', Arial, Helvetica, sans-serif;
+    font-size: 34px;
+    font-weight: 700;
+    padding: 36px 16px;
+    margin: 0;
+}
+.card__sub-content{
+    display: flex;
+    flex-direction: column;
+    padding: 20px;
+    align-items: flex-start;
+    justify-content: center;
+}
+.card__paragraphs{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+}
+.card__text{
+    color: #fff;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    line-height: 20px;
+    font-size: 16px;
```

Comment:
check font-size and line height here then check both on larger widths.

Reply:
Was looking over.
The customers card had a larger font / line-height compared to the other cards on mobile, but tablet and desktop were the same (which I did forget to add)